.github/actions/multi-approvers/tests/main.test.ts

```diff
@@ -62,4 +87,30 @@ test("#main", { concurrency: true }, async (suite) => {
       "Multi-approvers action failed: unexpected event [push].",
     );
   });
+
+  await suite.test(
+    "should fail if user-id-allowlist contains invalid numeric ID",
+    async (t) => {
+      const invalidAllowlistInput = "123,abc,456";
+      const core = newFakeCore({
+        token: "fake-token",
+        team: "fake-team",
+        "user-id-allowlist": invalidAllowlistInput,
+      });
+      const setFailed = t.mock.method(core, "setFailed", () => {});
+      const context = createFakePullRequestContext();
+
+      await main(core, context);
+
+      assert.equal(setFailed.mock.calls.length, 1);
+      const failMsg = String(setFailed.mock.calls[0].arguments[0]);
+
+      assert.ok(
+        failMsg.includes(
+          "Multi-approvers action failed: invalid allowlisted user ID: [abc].",
+        ),
+        `Expected error message to specify 'abc', but got: ${failMsg}`,
```

Comment:
I'd prefer to not have this. Using

```
assert.equal(failMsg, `Invalid allowlisted user ID: [abc]. Full input (user-id-allowlist): [123,abc,456]`);
```

should give an exact description of what went wrong (and then we don't need to add more code describing the problem).